### PR TITLE
feat(pci.projects.private-registry): add component for offer selection

### PR DESCRIPTION
**Pull request by marie-j**

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/private-registry/components/plan-upgrade/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "private_registry_upgrade_plan": "Changer de plan",
+  "private_registry_upgrade_plan_description": "Vous pouvez à tout moment migrer d'un plan à un plan plus important. La prise en compte des nouvelles limites est effective en quelques dizaines de secondes, sans perte de données ni changement de configuration nécessaire. Si vous créez, modifiez ou supprimez un service en cours de mois, vous serez facturé uniquement pour les heures effectives de votre plan. Le trafic est illimité et inclus, quelque soit sa destination.",
+  "private_registry_upgrade_plan_connections": "{{ total }} connexions simultanées ({{ concurrency }} * {{ connectionsNumber }})",
+  "private_registry_upgrade_plan_vulnerability": "Scan des vulnérabilités",
+  "private_registry_upgrade_plan_traffic": "100% SLA durable et traffic illimité"
```
Comment (JeremyDec) [inline packages/manager/modules/pci/src/projects/project/private-registry/components/plan-upgrade/translations/Messages_fr_FR.json]:
```suggestion
  "private_registry_upgrade_plan_traffic": "100% SLA durable et trafic illimité"
```

File: packages/manager/modules/pci/src/projects/project/private-registry/components/plan-upgrade/controller.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+import map from 'lodash/map';
+import sortBy from 'lodash/sortBy';
+import RegistryPlan from './RegistryPlan.class';
+
+export default class {
+  /* @ngInject */
+  constructor(
+    CucCloudMessage,
+    OvhApiOrderCatalogPublic,
+    OvhApiCloudProjectContainerRegistryPlan,
```
Comment (ganeshkumar1989) [inline packages/manager/modules/pci/src/projects/project/private-registry/components/plan-upgrade/controller.js]: Please order them alphabetically :)

**Comment by ovh-cds** [comment]
CDS Report build-monorepository#9153.0 ■   
* 
  * Build ■

**Comment by ovh-cds** [comment]
CDS Report test-manager_2#9153.0 ■   
* Stage 1
  * e2e -   
  * lint ■

**Comment by ovh-cds** [comment]
CDS Report test-manager_2#9169.0 ■   
* Stage 1
  * e2e ■   
  * lint ■

**Comment by ovh-cds** [comment]
CDS Report build-monorepository#9169.0 ■   
* 
  * Build ■

**Comment by ovh-cds** [comment]
CDS Report build-monorepository#9277.0 ■   
* 
  * Build ■

**Comment by ovh-cds** [comment]
CDS Report installation#9307.0 ■   
* Stage 1
  * Install ■

**Comment by ovh-cds** [comment]
CDS Report build-monorepository#9371.0 ✘  
* 
  * Build ✘

**Pull request merged**